.github/settings.yml

```diff
@@ -0,0 +1,4 @@
+_extends: .github
+repository:
+  name: eslint-config-scoop
+  private: false
```

Comment:
Will this still enforce the appropriate PR approval requirements? Am guessing we just want anyone in the Engineering org to be able to approve PR requests